### PR TITLE
Enable to annotate primitive types to endpoint

### PR DESCRIPTION
**Pull request by kobayang**
I want annotate primitive types like string to endpoints.
When use symbols, annotate those symbol types as follows.

```
class BooksController
  extend OpenApiAnnotator::ControllerAnnotatable

  endpoint [:string]
  def index
    # some code
  end
end
```

```
 "books":
    get:
      responses:
        '200':
          description: Returns an array of string
          content:
            application/json:
              schema:
                type: array
                items:
                  type: string
```

### CODE REVIEW

**Comment by kobayang** [comment]
@ngtk Please review

**Comment by ngtk** [comment]
@kobayang 

Oh, now we can not handle primitive types and these are treated as a custom component.
Sorry about that! 😨 

But, can we use a class instead of a symbol in order to specify the OpenAPI type? 💡 
Because of interface consistency, which means annotation should be ruby level, not OpenAPI spec.
Also, we can handle this because the granularity of the OpenAPI type is bigger than Ruby's one. but, if we want to handle the OpenAPI format, we should use a symbol.

But, for now, I think that it should be used Ruby class for the interface and annotator should convert a Ruby class to an OpenAPI type.

FYI: Definition of `type` and `format` https://swagger.io/docs/specification/data-models/data-types/

Anyway, thank you for pull requesting! 😄

**Comment by kobayang** [comment]
@ngtk 

Does this mean `endpoint [String]` is better than `endpoint [:string]` ?
But we can not know the String class is customized or not.

Or can we use OpenApi::DataType class like `endpoint [OpenApi::DateType(:string, :string)]` ?
cf. https://github.com/ngtk/open_api/blob/master/lib/open_api/data_types.rb

**Comment by ngtk** [comment]
> Does this mean endpoint [String] is better than endpoint [:string] ?
> But we can not know the String class is customized or not.

What do you mean "customized"? Do you mean "user-defined"? or "Singular class"?
Anyway, I think we can convert `String` to `OpenApi::DateType(:string, :string)` in this library.

**Comment by kobayang** [comment]
f279d71 Enable to annotate some primitive ruby classes (String, Integer, and Float).
cf.  https://github.com/ngtk/open_api/blob/master/lib/open_api/data_types.rb

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ language: ruby
 rvm:
   - 2.4.4
   - 2.5.1
-before_install: gem install bundler -v 1.16.1
+before_install:
+  - gem install -v 1.17.3 bundler --no-rdoc --no-ri
+  - bundle _1.17.3_ install
```
Comment (ngtk) [inline .travis.yml:11]: Thanks!

**Review by ngtk — APPROVED** [review]
Looks great! 👍✨

**Comment by ngtk** [comment]
Thank you for contributing! 🎉
This change has been released in `v0.4.0`! https://rubygems.org/gems/open_api_annotator

**Pull request merged**